seqr/views/utils/test_utils.py

```diff
@@ -516,11 +514,43 @@ def get_group_members_side_effect(user, group, use_sa_credentials=False):
     return {}
 
 
-class AnvilAuthenticationTestCase(AuthenticationTestCase):
+class DifferentDbTransactionSupportMixin(object):
```

Comment:
this mixin allows us to use transactions for the postgres database but not for clickhouse - django by default forces an either/or, and transactions in postgres greatly speed up our tests